Sources/core/objects/Interface.cpp

```diff
@@ -86,6 +92,19 @@ Slot<Item>& HumanInterface::GetActiveHand()
         return l_hand_;
 }
 
+void HumanInterface::UpdatePulling(bool is_pulling)
+{
+    pulling_ = is_pulling;
+}
+
+void HumanInterface::StopPulling()
```

Comment:
Better to name it just `StopPull` (because it is proxy function, so it is kind of same with `StopPull` in the `Human` class, so it will be easier to understand what's going on in this code).
Now someone can think that `StopPulling` do something directly with `pulling_` variable (like `UpdatePulling`), but it is not true.